### PR TITLE
Lewis navbar update

### PR DESCRIPTION
**Pull request by lewis-conroy**
third time is the charm.

Added media queries to scale the font down for responsiveness.
Fixed the styling and href for the logo as well.

Looking good to me :)

### CODE REVIEW

**Comment by lewis-conroy** [comment]
Just realised it's still missing the url for the donation.

**Comment by arthurHarding** [comment]
On the homepage, the navbar is the same colour as the hero, which I'm hoping to achieve by having a transparent nav bar on the home page. Is that possible?

File: CooperationHullMainSite/Views/Shared/_MainNavBar.cshtml
Comment (arthurHarding) [inline CooperationHullMainSite/Views/Shared/_MainNavBar.cshtml:1]: can this whole file be deleted?

**Comment by lewis-conroy** [comment]
> On the homepage, the navbar is the same colour as the hero, which I'm hoping to achieve by having a transparent nav bar on the home page. Is that possible?

aye I can do 👍

**Review by arthurHarding — APPROVED** [review]

**Review by arthurHarding — APPROVED** [review]

**Pull request merged**